build.cmd

```diff
@@ -1,15 +1,8 @@
-@echo off
-
-if "%~1"=="-h" goto help
-if "%~1"=="-help" goto help
-if "%~1"=="-?" goto help
-if "%~1"=="/?" goto help
-
-powershell -ExecutionPolicy ByPass -NoProfile -File "%~dp0eng\common\build.ps1" %*
-goto end
-
-:help
-powershell -ExecutionPolicy ByPass -NoProfile -Command "& { . '%~dp0eng\common\build.ps1'; Get-Help }"
-
-:end
-exit /b %ERRORLEVEL%
+@echo off
+setlocal
+
+set _args=%*
+if "%~1"=="-?" set _args=-help
+
+powershell -ExecutionPolicy ByPass -NoProfile -File "%~dp0eng\build.ps1" %_args%
+exit /b %ERRORLEVEL%

```

Comment:
How is this change related? It looks fine to me on paper but the Windows build CI failed.

Reply:
Oh, I see, for consistency with build.sh.